### ISSUE TITLE
ActionPopover  : CSS to review for ActionPopver   

### ISSUE THREAD

**Issue by hagad** [Sage/carbon#3455]
### Current behaviour
After exchange with the UX.
UX want to change the CSS of 3 dots  and the menu of the ActionPopver  component 

### Expected behaviour
When the action popover is in an active state, the background colour of the 3 dots hit area should turn Slate grey 70 (see design spec). This is to help the user identify which button relates to the menu that is currently active as there is currently no active state for this in carbon.

![image](https://user-images.githubusercontent.com/2328042/101060851-c1032d00-3587-11eb-9141-f019f70a07dd.png)

### Reproducible example

https://carbon.sage.com/?path=/docs/design-system-flat-table--paginated-with-sticky-header

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| carbon-react     | v50.3.0
| carbon-factory   |
| react-scripts    | 
| React            |
| Browser          |
| npm              |
| Operating System |

**Comment by jamime** [comment]
I asked for more information on Slack.

**Comment by lukeemmerson-finch** [comment]
@edleeks87 copied over the ticket info into this git issue.

**Comment by jamime** [comment]
Conversation ongoing in slack.

**Comment by jamime** [comment]
Here is an example of what is available today using `FlatTable` using the `hightlighted` prop and`ActionPopover`.
https://codesandbox.io/s/flat-table-action-popover-3455-1l0ds?file=/src/index.js
![3455](https://user-images.githubusercontent.com/2328042/101290033-d67e8e00-37f7-11eb-9b65-e84780db6ba8.gif)

**Comment by jamime** [comment]
I had a call with @hagad and @lukeemmerson-finch.

The original issue stemmed from the drop shadow on the menu being opened made it appear like the popover was not associated with the button that opened it when the placement is `top`.

![image](https://user-images.githubusercontent.com/2328042/101344847-0bcdbf00-387e-11eb-97dd-f9fd3365eb76.png)

For comparison this is how it looks when placement is `bottom`.

![image](https://user-images.githubusercontent.com/2328042/101344917-30299b80-387e-11eb-99ab-41f7888c30ea.png)


@lukeemmerson-finch highlighted that there was no real affordance to show which row opened the popover which is where these updated designs came from.

I provided an example showing how @hagad could use the `highlighted` prop so the row remains highlighted whilst the action popover is open. We decided that we are not going to change the drop shadow, or the menu button color and @hagad is going to implement the `hightlighted` state.
